spec/schemas/_core.msearch.yaml

```diff
@@ -72,23 +72,23 @@ components:
                 $ref: '_common.query_dsl.yaml#/components/schemas/KnnQuery'
         from:
           description: |-
-            Starting document offset. By default, you cannot page through more than 10,000
+            The starting document offset. By default, you cannot page through more than 10,000
             hits using the from and size parameters. To page through more hits, use the
-            search_after parameter.
+            `search_after` parameter.
           type: number
         highlight:
           $ref: '_core.search.yaml#/components/schemas/Highlight'
         indices_boost:
-          description: Boosts the _score of documents from specified indices.
+          description: Boosts the `_score` of documents from specified indexes.
           type: array
           items:
             type: object
             additionalProperties:
               type: number
         min_score:
           description: |-
-            Minimum _score for matching documents. Documents with a lower _score are
-            not included in the search results.
+            The minimum `_score` for matching documents. Documents with a lower `_score` than the minimum
```

Comment:
Do we mean "The minimum `_score` required for document matching" or "The minimum `_score` required for documents to match the [noun]"?

Reply:
The first.